Add security-bundle test (#303)
giantswarm/cluster-test-suites: CHANGELOG.md

## [Unreleased]

### Added

> - Add `security-bundle` test in the common/apps.go file.

Reviewer: ```suggestion
- Wait and test for `security-bundle` apps to be deployed.
```
